### PR TITLE
Updated onboarding page to support new Quest and transcript upload

### PR DESCRIPTION
**Pull request by derrekchow**
Quest has recently been horribly redesigned and the method to upload transcripts has changed (see #334 and #336)

This change updates the onboarding page instructions and video to match the new Quest and transcript upload method. It is also presented in the form of a cleaner and simpler UI by the removing the laptop image and creating more equal visual importance between the video and the instructions.

## Before:
image

## After:
image

### CODE REVIEW

File: server/templates/onboarding_page.html [hunk 1]
```diff
@@ -11,44 +11,43 @@
 {% block body %}
   <div class="row-fluid">
     <div class="span12" id="onboarding-container">
-
+      <h2>Thanks for signing up!</h2>
```
Comment (AyushK1) [inline server/templates/onboarding_page.html]: Two exclamation marks in two successive sentences seems a little excessive; I propose:
Thanks for signing up! ... ! -> Thanks for signing up. ... !

**Comment by AyushK1** [comment]
Nice design decision! The image conveys the intended meaning while looking less cluttered. 👍

**Comment by dshynkev** [comment]
Also, can we host the video externally while we're at it?

**Comment by derrekchow** [comment]
> Also, can we host the video externally while we're at it?

**Comment by derrekchow** [comment]
.

**Comment by derrekchow** [comment]
> .

**Pull request merged**